src/BlockTypesController.php

```diff
@@ -201,6 +201,7 @@ protected function get_block_types() {
 		if ( Package::feature()->is_feature_plugin_build() ) {
 			$block_types[] = 'Checkout';
 			$block_types[] = 'Cart';
+			$block_types[] = 'CartOrderSummaryTaxesBlock';
```

Comment:
We might want to have this logic inside Cart.php class?

Reply:
✅ Fixed in: [83a8b43](https://github.com/woocommerce/woocommerce-blocks/pull/6737/commits/83a8b433fe749017f330920db89ab70501426037)